tech/languages/dotnet/about.md

```diff
@@ -0,0 +1,35 @@
+---
+title: .NET
+subsection: dotnet
+section: tech-languages
+order: 1
+version: 7.8.4
+description: ".NET Core and Mono are open-source .NET platforms. .NET applications are developed using the C#, F# and VB.NET programming languages."
+---
+
+.NET Core and Mono are available on Fedora. .NET applications are developed using the C#, F# and VB.NET programming languages.
+
+## .NET Core
+
+[.NET Core](https://docs.microsoft.com/en-us/dotnet/core/) is a general-purpose, modular, cross-platform and open-source development Platform.
+
+To install .NET Core, add the .NET SIG's [copr](http://developer.fedorainfracloud.org/deployment/copr/about.html) repository and install the `dotnet` package.
+
+```
+$ sudo dnf copr enable -y @dotnet-sig/dotnet
+$ sudo dnf install dotnet
```

Comment:
As `dotnet.md`  page already contains this, it seems like is duplicating content. Could you just link to it with the text below?